src/tutorials/Eljen-Background-Characterization.md

```diff
@@ -0,0 +1,342 @@
+<!-- #endregion -->
+
+```python id="ouxA8rpoxfUB"
+# Function to update the plot based on Z
+def plot_outliers(Z):
+    threshold_poisson = lambda_ + Z * np.sqrt(lambda_)
+    poisson_outliers = neutron_df_1_minute_background[neutron_df_1_minute_background["Counts"] > threshold_poisson]
+
+    plt.figure(figsize=(14, 5))
+    plt.plot(neutron_df_1_minute_background.index, neutron_df_1_minute_background["Counts"],
+             label="Counts per Minute", color='blue', linewidth=1)
+
+    plt.scatter(poisson_outliers.index, poisson_outliers["Counts"],
+                color='red', label=f"Outliers (> {threshold_poisson:.2f} counts)", zorder=5)
+
+    plt.axhline(threshold_poisson, color='gray', linestyle='--', linewidth=1.5,
+                label=f"Z = {Z:.1f} Threshold")
+
+    plt.title(f"Neutron Count Time Series with Z={Z:.1f} Outlier Threshold")
+    plt.xlabel("Time")
+    plt.ylabel("Counts per Minute")
+    plt.legend()
+    plt.grid(True)
+    plt.tight_layout()
+    plt.show()
+```
+
+<!-- #region id="LYRh5-UrRJlO" -->
+In order to visualize the effect of choosing different values of $Z$, feel free t interact with the code block below and play around with the variables!
+<!-- #endregion -->
+
+```python colab={"base_uri": "https://localhost:8080/", "height": 364} id="X6bom6B1Q97q" outputId="c8b88576-8c3e-4b29-9371-0fc09a204c14"
+plot_outliers(Z=3)
+```
+
+<!-- #region id="Kno9Dvjc4H3i" -->
+Let us now perform the same analysis for gamma counts.
+<!-- #endregion -->
+
+```python id="K3sVmWq1yWUu"
+# Precompute Poisson mean for gamma background
+lambda_gamma = gamma_df_1_minute_background["Counts"].mean()
+
+# Define the plotting function
+def plot_gamma_outliers(Z):
+    threshold_poisson = lambda_gamma + Z * np.sqrt(lambda_gamma)
+    poisson_outliers = gamma_df_1_minute_background[gamma_df_1_minute_background["Counts"] > threshold_poisson]
+
+    plt.figure(figsize=(14, 5))
+    plt.plot(gamma_df_1_minute_background.index, gamma_df_1_minute_background["Counts"],
+             label="Gamma Counts per Minute", color='purple', linewidth=1)
+
+    plt.scatter(poisson_outliers.index, poisson_outliers["Counts"],
+                color='orange', label=f"{len(poisson_outliers)} Outliers (> {threshold_poisson:.2f} counts)", zorder=5)
+
+    plt.axhline(threshold_poisson, color='gray', linestyle='--', linewidth=1.5,
+                label=f"Z = {Z} Threshold")
+
+    plt.title(f"Gamma Count Time Series with Z={Z} Outlier Threshold")
+    plt.xlabel("Time")
+    plt.ylabel("Gamma Counts per Minute")
+    plt.legend()
+    plt.grid(True)
+    plt.tight_layout()
+    plt.show()
+```
+
+<!-- #region id="xmPhCdfGRWM8" -->
+Similarly to above, feel free to play around with $Z$ to observe the effect of its value on our count threshold.
+<!-- #endregion -->
+
+```python id="NSK1Nxi67eqi" colab={"base_uri": "https://localhost:8080/", "height": 364} outputId="6e86efc6-a642-4b80-fc3a-5bef65a559d4"
+plot_gamma_outliers(Z=3)
+```
```

Comment:
Finishing with a summary of what's been done / how can people use this etc would be a nice to have.